backend/utils/booking-utils.js

```diff
@@ -0,0 +1,35 @@
+const mailFormat = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```

Comment:
Import it from utils.

Reply:
I separated the utils file according to controllers, booking-utils/company-utils/service-utils

Reply:
Right, but the principle of DRY (Don't Repeat Yourself) is broken since there is a copy of it in the utils file.